core/src/main/java/de/jplag/highlightextraction/CompleteMatchesStrategy.java

```diff
@@ -0,0 +1,26 @@
+package de.jplag.highlightextraction;
+
+import static de.jplag.highlightextraction.SubSequenceUtil.addSequence;
```

Comment:
Also, try to replace the static import with a non-static import.
Then call `SubSequenceUtil.addSequence()`.